polaris-react/src/components/Bleed/Bleed.stories.tsx

```diff
@@ -42,7 +42,7 @@ export function Default() {
 
 export function WithVerticalDirection() {
   return (
-    <Box background="bg" padding="4" border="base">
+    <Box background="bg" padding="4" borderColor="border" borderWidth="1">
```

Comment:
```suggestion
    <Box background="bg" padding="4" borderColor="border-subdued" borderWidth="1">
```
[`base` should map to `border-subdued`](https://github.com/Shopify/polaris/blob/main/polaris-migrator/src/migrations/v11-styles-replace-custom-property-border/v11-styles-replace-custom-property-border.ts#L10-L11)